frontend/src/modules/organization/components/organization-identities.vue

```diff
@@ -12,11 +12,12 @@
           :platform="platform"
           :username-handles="getHandlesByPlatform(platform)"
           :links="getUrlsByPlatform(platform)"
+          :href="getUrlsByPlatform(platform)[0]"
           :track-event-name="getPlatformDetails(platform)?.trackEventName"
           :track-event-channel="getPlatformDetails(platform)?.trackEventChannel"
           :tooltip-label="getPlatformDetails(platform)?.tooltipLabel"
           :show-handles-badge="true"
-          :as-link="getUrlsByPlatform(platform).length ? getUrlsByPlatform(platform)[0] : false"
+          :as-link="getPlatformDetails(platform).asLink"
```

Comment:
I just double checked, and if organization identity does not have a `url`, we are redirecting to https://github.com/github.  However, when there's no url in identity we should not show button as a link.
Could you fix this